src/apps/replication/meta_server/server_state.cpp

```diff
@@ -503,6 +531,8 @@ void server_state::update_configuration(
             std::stringstream ss;
             ss << "/apps/" << app.app_name << "/" << old.gpid.pidx;
             partition_path = ss.str();
+            // TODO(qinzuoyan): last_drops is maintained only in meta_server and nerver used in replica_server,
+            // consider take it out of partition_configuration to avoid unnecessary data transmission.
```

Comment:
we are considering another faster proposal method for adding secondary, done by primary, which requires the drops data structure known on primary. so let's keep this field in the data structure.